src/test/java/io/jenkins/plugins/explain_error/ConsoleExplainErrorActionTest.java

```diff
@@ -92,158 +97,135 @@ void testCreateCachedResponseWithLongExplanation() throws Exception {
 
     @Test
     void testCreateCachedResponseWithSpecialCharacters() throws Exception {
-        // Use reflection to access the private method
-        Method method = ConsoleExplainErrorAction.class.getDeclaredMethod("createCachedResponse", String.class);
-        method.setAccessible(true);
-
         String specialExplanation = "Error with special chars: <>&\"'\nUnicode: ñáéíóú 中文 العربية";
-        String cachedResponse = (String) method.invoke(action, specialExplanation);
+        String cachedResponse = action.createCachedResponse(specialExplanation);
 
         assertNotNull(cachedResponse);
         assertTrue(cachedResponse.contains(specialExplanation));
         assertTrue(cachedResponse.contains("previously generated explanation"));
     }
 
     @Test
-    void testExistingExplanationDetection() {
```

Comment:
All the removed tests didn't test anything from the `ConsoleExplainErrorAction` class. What they tested is already covered in `ErrorExplanationActionTest`